### DISCUSSION TITLE
Feature request: allow no Element Index sources (without breaking the page)

### DISCUSSION THREAD

**Discussion by bertoost** [craftcms/cms#8593] [Ideas]
### Description

When an element index page just has the wildcard (all) source, it should be possible to hide the sidebar. 

Or do this automatically when you just have none or one source defined. Since this is weird. It does nothing and probably when you just have one source, you don't want a feature-less sidebar.

Or even don't define an element source when you don't want them.

Related stackexchange:
https://craftcms.stackexchange.com/questions/20440/elementtype-cp-section-does-not-display-table

### Additional info

- Craft version: 3.0.24
- PHP version: 7.1.20

**Comment by Anubarak** [comment]
If you just want a quick fix to hide them you can just insert your own css into the template or overwrite it since you already extend the correct template.

I created a field type that renders an `elementIndex` to show all related entries for an entry and did the same
